app/models/referenceData/CustomsOffice.scala

```diff
@@ -18,7 +18,10 @@ package models.referenceData
 
 import play.api.libs.json.{Json, OFormat}
 
-case class CustomsOffice(id: String, name: String, roles: Seq[String], phoneNumber: Option[String])
+case class CustomsOffice(id: String, name: String, phoneNumber: Option[String]) {
+  val nameOption  = if (name.isEmpty) None else Some(name)
+  val phoneOption = if (phoneNumber.isDefined && !phoneNumber.get.isEmpty) Some(phoneNumber.get) else None
+}
```

Comment:
phoneNumber.filter(_.nonEmpty)?

Returns this Option if it is nonempty and applying the predicate p to this Option's value returns true. Otherwise, return None.
This is equivalent to:
 option match {
   case Some(x) if p(x) => Some(x)
   case _               => None
 }
 
Params:
p – the predicate used for testing.